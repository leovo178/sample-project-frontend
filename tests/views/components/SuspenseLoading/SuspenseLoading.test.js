import React from 'react'

import { cleanup, render } from '@testing-library/react'

import { SuspenseLoading } from '@views_components'

describe('SuspenseLoading', () => {
	afterEach(() => {
		cleanup()
	})

	it('should match snapshot', () => {
		const { container, queryByText } = render(<SuspenseLoading />)

		expect(container).toMatchSnapshot()
		expect(queryByText('Loading...')).toBeTruthy()
	})
})
