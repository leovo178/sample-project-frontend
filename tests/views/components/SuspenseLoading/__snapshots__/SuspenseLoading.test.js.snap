// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SuspenseLoading /> should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-3 makeStyles-root-1"
  >
    <div
      class="makeStyles-root-4"
    >
      Loading...
    </div>
  </div>
</div>
`;
